### PR TITLE
Updated faucet plugin to use the new endpoint

### PR DESCRIPTION
**Pull request by Bullrich**
Updated the faucet plugin to use the new endpoint: `/drip/web`

This commit will close https://github.com/paritytech/substrate-matrix-faucet/issues/215

The reason for which we updated the endpoint is that the previous endpoints switches between the bot and the docs, and with the new change the endpoint will exclusively work with the web removing all of the code bot. You can find the commit here: https://github.com/paritytech/substrate-matrix-faucet/commit/56130249044c67d12fa9a434c13314d032d55d22

@rzadp has helped me test it. [This is his advice on how to do so](https://github.com/paritytech/ink-docs/pull/145#issuecomment-1423869427)

### CODE REVIEW

**Review by cmichi — APPROVED** [review]
Thank you!

**Comment by cmichi** [comment]
Oh @Bullrich Can you update it here too? https://github.com/paritytech/ink-docs/blob/master/i18n/es/docusaurus-plugin-content-docs/current/testnet/Faucet.tsx

It's not ideal, but we just copied the file for the Spanish website part.

**Comment by Bullrich** [comment]
> Oh @Bullrich Can you update it here too? https://github.com/paritytech/ink-docs/blob/master/i18n/es/docusaurus-plugin-content-docs/current/testnet/Faucet.tsx

@cmichi I updated the spanish link and I also took the liberty of translating the alt text to spanish. I'll wait for your approval before merging.

**Review by cmichi — APPROVED** [review]

**Pull request merged**
